sirepo/package_data/static/js/activait.js

```diff
@@ -249,6 +263,14 @@ SIREPO.app.directive('appHeader', function(appState, activaitService) {
                 }
                 return false;
             };
+            $scope.isImageToImage = () => {
+                var info = appState.models.columnInfo;
+                if (info && info.inputOutput.indexOf('output') >= 0) {
+                    var idx = info.inputOutput.indexOf('output');
+                    return info.shape[idx].slice(1, info.shape[idx].length).length > 1;
+                }
```

Comment:
When viewing the "2019 World Happiness" example, I'm seeing a "cannot read properites of undefined" in isImageToImage()